include/mbedtls/aes.h

```diff
@@ -69,26 +69,49 @@ extern "C" {
 // Regular implementation
 //
 
+/** VIA padlock needs 16 extra Bytes for alignment. */
+#if (defined(MBEDTLS_PADLOCK_C) && defined(MBEDTLS_HAVE_X86)) ||        \
+    (defined(MBEDTLS_AESNI_C) && MBEDTLS_AESNI_HAVE_CODE == 2)
+#define MBEDTLS_PADLOCK_EXTRA 4
+#else
+#define MBEDTLS_PADLOCK_EXTRA 0
+#endif
+
+/** Each round key occupies 16 Bytes. */
+#define MBEDTLS_AES_RK_UINT32S 4
```

Comment:
Abstracting this here makes things less readable, not more. To review or even read I need to go from the definition of `buf` up to here and back, which completely stops fluent reading

Maybe the total in the `buf` definition should be `xxx_BYTES`, then it can be `/ sizeof(uint32_t)`